starter_code/routes/movies.js

```diff
@@ -0,0 +1,22 @@
+const express = require("express");
+const router = express.Router();
+const Movie = require("../models/Movie");
+
+router.get("/movies", (req, res) => {
+  // get all the movies
+  Movie.find().then((movies) => {
+    // render a 'movies' view with the movies data
+    res.render("movies", { moviesList: movies });
+  });
+  // res.send('this is the movies route');
+});
+
+router.get("/movies/:movieId", (req, res) => {
+  const movieId = req.params.movieId;
+  Movie.findById(movieId).then((movie) => {
+    // res.send(movie);
+    res.render("movieDetails", { movie: movie });

```

Comment:
Ver small thing, in modern Javascript you can make this line a little smaller :)

```js
res.render("movieDetails", {movie})
```